systemtest/src/test/java/io/strimzi/systemtest/StrimziUpgradeST.java

```diff
@@ -342,6 +343,12 @@ void deployClients(String image, KafkaUser kafkaUser) {
             .endSpec().done();
     }
 
+    void assertNoCoErrorsLogged(long sinceSeconds) {
```

Comment:
Are you sure it's used only in upgrade?

Reply:
Och, mistake were made...the method is in `BaseST`. I will remove it from StrimziUpgradeST. Thanks.